src/main/java/frc/robot/commands/ShooterEject.java

```diff
@@ -41,6 +43,7 @@ public void execute() {
         m_CDSSubsystem.CDSBeltToggle(false);
         m_ShooterSubsystem.runCargo(Constants.Shooter.cargoForward);
         m_ShooterSubsystem.setCargoBoolean(true);
+        Shuffleboard.addEventMarker("Fired Cargo", EventImportance.kHigh);
```

Comment:
There are other places where cargo is fired.  Would be good to consider if we want to add events to each of the other commands.  

Or, check and see if "setCargoBoolean" is used anywhere else.  Because it might be better to set the "Fired Cargo" event in a method that all three call, as long as it's not used somewhere else

Reply:
Changed it to where it does that when it fires rather than where it tells the CDS to funnel another ball up.